packages/lwc/__tests__/isomorphic-exports.spec.ts

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2024, Salesforce, Inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
+ */
+import * as engineDom from '@lwc/engine-dom';
+import * as engineServer from '@lwc/engine-server';
+import * as ssrRuntime from '@lwc/ssr-runtime';
+
+describe('isomorphic package exports', () => {
```

Comment:
@nolanlawson did I get these the right way around?

Reply:
Yep this looks right!